### PR TITLE
add: utilities to typocomp

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
add: utilities classes
    add: diff sizes font 
    add: headings
Screenshot 2022-02-04 at 6 04 07 PM

### CODE REVIEW

**Review by kirti2602 — APPROVED** [review]
Looks good to me. One suggestion though- The code `iframe`s are getting cut in mobile view for me, so they can be improved if you're concerned about mobile view now

**Review by sakshi006 — COMMENTED** [review]
Good to go.

File: components/typography/typo.css [hunk 1]
```diff
@@ -21,7 +21,36 @@ p {
...
+}
+.text-lg {
+  font-size: 1.2rem;
+}
+/* heading */
+.heading-lg {
+  font-size: 1.5rem;
+  font-weight: 400;
+}
+.heading-x-lg {
```
Comment (AllanSam-codes) [inline components/typography/typo.css:41]: Prefer two words for naming classes

**Review by aneenajohn — CHANGES_REQUESTED** [review]
Component Library looks pretty clean.
Few things to take care :
1) Be consistent with class names -> Classname-modifier eg: btn-sm
2) Stick with 1/2 letter naming instead 3 or more.
3) Add variable names for white and black as well
4) Add active state on the side nav bar.

**Pull request merged**